test/runtime/samples/element-source-location/_config.js

```diff
@@ -11,14 +11,14 @@ export default {
 
 		assert.deepEqual(h1.__svelte_meta.loc, {
 			file: path.relative(process.cwd(), path.resolve(__dirname, 'main.svelte')),
-			line: 4,
+			line: 5,
 			column: 0,
 			char: 53
 		});
 
 		assert.deepEqual(p.__svelte_meta.loc, {
 			file: path.relative(process.cwd(), path.resolve(__dirname, 'Foo.svelte')),
-			line: 1,
+			line: 2,
```

Comment:
same for the p, the previous combination of line 1, col 1 and pos 7  would require cramming 6 chars at col 0.